package.json

```diff
@@ -23,6 +23,7 @@
     "@typescript-eslint/eslint-plugin": "^5.22.0",
     "@typescript-eslint/eslint-plugin-tslint": "^5.22.0",
     "@typescript-eslint/parser": "^5.22.0",
+    "core-js": "^2.6.12",
```

Comment:
The latest version of `core-js` is 3.6.5. Please consider to use it.

Reply:
Right, updated in 88ed21b